--- boost_1_57_0/boost/range/detail/any_iterator.hpp	2014-08-05 06:06:25.000000000 +0900
+++ ./boost/range/detail/any_iterator.hpp	2015-02-05 14:28:23.447064025 +0900
@@ -114,6 +114,7 @@
         };
     } // namespace range_detail
 
+    namespace iterators {
     namespace detail
     {
         // Rationale:
@@ -246,7 +247,8 @@
         };
 
 
-    }
+    } // namespace dettail
+    } // namespace iterators
 
     namespace range_detail
     {
